src/views/LoginSuccess.vue

```diff
@@ -36,9 +36,12 @@ import { WalletStore } from '@/lib/WalletStore';
 import staticStore, { Static } from '@/lib/StaticStore';
 import { PageHeader, PageBody, LabelInput, AccountList, PageFooter, SmallPage } from '@nimiq/vue-components';
 import Network from '@/components/Network.vue';
+import AccountFinder from '../lib/AccountFinder';
 
 @Component({components: {PageHeader, PageBody, LabelInput, AccountList, Network, PageFooter, SmallPage}})
 export default class LoginSuccess extends Vue {
+    private static readonly DEFAULT_ACCOUNT_LABEL: string = 'Standard Account';
```

Comment:
I"d say lets think about where to put those kind of constants. It gets used in other places too, so we should have a collection of them somehow.

Reply:
Tracked in https://github.com/nimiq/accounts/issues/86.